src/dynamic_graph_head/sim_head.py

```diff
@@ -6,6 +6,7 @@
 All rights reserved.
 """
 
+from os import write
```

Comment:
Why do you import the `write` function?

Reply:
i don't even remember adding this, removed it